packages/parser/src/ASTParser.ts

```diff
@@ -184,6 +184,15 @@ class ASTParser {
               ASTParser.logEntry(`[Info] Open jsxElement`);
               jsxElement.open(node);
             } else {
+              if (node.name.type === 'JSXIdentifier') {
+                if (
+                  jsxElement.isRoute() &&
+                  attribute.getElementName() == 'element'
```

Comment:
I think you are on the right track here, although I do not think it is necessary with the if statement `node.name.type === 'JSxIdentifier'`

Is there a specific reason for this?